app/components/check_your_answers_summary_component.rb

```diff
@@ -0,0 +1,37 @@
+class CheckYourAnswersSummaryComponent < ViewComponent::Base
```

Comment:
Is this component check your answers specific? Could it be a generic summary card component?

Reply:
Perhaps `CheckYourAnswers` is the wrong prefix, but I wanted to make it clear that because it accepts a `model`, retrieves the values from it and hard codes a "Change" link, it's not _quite_ generic enough to just be a summary card component (i.e. if it was going to be generic I'd have built it in `govuk-components` and have the same API as the `govuk_summary_list` helper).

Either way, we need something which wraps the summary list card component and does the specific logic this component does.

Perhaps `ModelSummaryComponent` is good enough?

Reply:
I see, in that case, keep it like this :+1: